src/OrchardCore.Modules/Orchard.OpenId/Controllers/AccessController.cs

```diff
@@ -24,6 +26,7 @@
 namespace Orchard.OpenId.Controllers
 {
     [Authorize]
+    [EnableCors(Constants.OpenIdConnectPolicy)]
```

Comment:
I agree with @anoordende. We need to make sure CORS is only applied to non-interactive endpoints, which excludes both the authorization and logout endpoints.